### PR TITLE
Ensembl transcript nested pfam domains

### PR DESCRIPTION
**Pull request by inodb**
Only last commit is relevant (dependent on #93 ):

Changes the ensembl transcript model to include pfamDomains field:
```
{
    "geneId": "string",
    "pfamDomains": [
      {
        "pfamDomainEnd": 0,
        "pfamDomainId": "string",
        "pfamDomainStart": 0
      }
    ],
    "proteinId": "string",
    "proteinLength": 0,
    "transcriptId": "string"
  }
```

### CODE REVIEW

File: web/src/main/resources/application.properties.EXAMPLE [hunk 1]
```diff
@@ -18,20 +18,12 @@ vep.isoform.overrides=uniprot:isoform_overrides_uniprot.txt,mskcc:isoform_overri
 # PDB header web service URL
 pdb.header_service_url=http://files.rcsb.org/header/PDB_ID.pdb
 
-# Ensembl biomart PFAM filename
-ensembl.biomart.pfam=ensembl_biomart_pfam.txt
```
Comment (onursumer) [inline web/src/main/resources/application.properties.EXAMPLE]: We may still want to keep these file names, and replace the hard-coded versions in the `GenomeNexusImporter`.

File: model/src/main/java/org/cbioportal/genome_nexus/model/PfamDomainRange.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
+package org.cbioportal.genome_nexus.model;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+
+public class PfamDomainRange
```
Comment (onursumer) [inline model/src/main/java/org/cbioportal/genome_nexus/model/PfamDomainRange.java]: we should also add a mixin under `web` module for this model

File: web/src/main/resources/ensembl_biomart_transcripts.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{"transcript_stable_id":"ENST00000000233","gene_stable_id":"ENSG00000004059","protein_stable_id":"ENSP00000000233","protein_length":180.0,"domains":[{"pfam_domain_id":"PF00025","pfam_domain_start":7.0,"pfam_domain_end":176.0}]}
```
Comment (onursumer) [inline web/src/main/resources/ensembl_biomart_transcripts.json]: Let's not forget to update this file with the full data :)

File: web/src/main/resources/application.properties.EXAMPLE (hunk 1 above)
Comment (inodb) [reply web/src/main/resources/application.properties.EXAMPLE]: @onursumer I'm still not sure using `GenomeNexusImporter` is the right way to initialize the database. Making it configurable at this point seems like premature optimization

**Pull request merged**